### PR TITLE
Extend release action to automatically upload artifacts to release.

### PR DESCRIPTION
**Pull request by Mpdreamz**
Uploads the profiler and startup hooks from the linux build.

Adds a windows build after the full release runs to also attach the
windows profiler build to the release.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -76,3 +81,23 @@ jobs:
         message: |
           :large_yellow_circle: [${{ github.repository }}] Release *${{ github.ref_name }}* could not be published.
           Build: (<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|here>)
+          
+  release-windows:
+    runs-on: ubuntu-latest
```
Comment (stevejgordon) [inline .github/workflows/release.yml:86]: Should this run on a Windows image?

Comment (Mpdreamz) [reply .github/workflows/release.yml:86]: Doh!

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -76,3 +81,23 @@ jobs:
...
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Bootstrap Action Workspace
+        uses: ./.github/workflows/bootstrap
+        with:
+          rust: 'true'
+
+      - name: Package
+        run: ./build.sh pack
```
Comment (stevejgordon) [inline .github/workflows/release.yml]: Do we need to switch to build.bat here now

**Review by stevejgordon — APPROVED** [review]
LGTM

**Pull request merged**